ansible/configs/rhel-security/audit.yml

```diff
@@ -8,16 +9,16 @@
     state: directory
     owner: auditlab
     group: auditlab
-    mode: '0700'
+    mode: u=rwx,go=
```

Comment:
that's really a matter of taste, Unix octal representation is second nature to a lot of people :)

Reply:
It is to me too, but I attribute it to some sort of brain damage I suffered :wink: